contrib/NeonVM/global.c

```diff
@@ -123,6 +137,23 @@ void ord(TStack *stack)
 
 
 
+
+void io_fprint(TStack *stack)
+{
+    Cell *s = peek(stack, 0);
+    Cell *pf = peek(stack, 1);
+
+    //FILE *f = check_file(pf);
+    FILE *f = check_file(stderr);
```

Comment:
Make sure you mark this with a pretty clear TODO comment!

Reply:
Fixed.